src/core_ocean/shared/mpas_ocn_vel_pressure_grad.F

```diff
@@ -537,6 +540,30 @@ subroutine ocn_vel_pressure_grad_tend(ssh, pressure, &
          !$acc exit data delete (JacobianDxDs, JacobianTz, JacobianSz)
          deallocate(JacobianDxDs, JacobianTz, JacobianSz)
 
+      case (pGradTypeConstForced)
+
+#ifdef MPAS_OPENACC
+         !$acc parallel loop &
+         !$acc    present(angleEdge,maxLevelEdgeTop, tend, edgeMask)
+         !$acc    private(pGrad)
+#else
+         !$omp parallel
+         !$omp do schedule(runtime) &
+         !$omp    private(pGrad)
+#endif
```

Comment:
@mattdturner I followed the examples above here. Please check.

Reply:
This should be the following
```
#ifdef MPAS_OPENACC
         !$acc parallel loop &
         !$acc    present(angleEdge,maxLevelEdgeTop, tend, edgeMask) &
         !$acc    private(pGrad, k)
#else
         !$omp parallel
         !$omp do schedule(runtime) &
         !$omp    private(pGrad, k)
#endif
```

I can't **bold** the changes in a codeblock, but the `acc    present` line needs an `&` at the end, and both the OpenMP and OpenACC `private` clauses should have `k` added.

Reply:
done, thanks.